### PR TITLE
[DOC] add installation req

### PR DESCRIPTION
**Pull request by tiandiao123**
## 📌 Checklist before creating the PR

- [ ] I have created an issue for this PR for traceability
- [ ] The title follows the standard format: `[doc/gemini/tensor/...]: A concise description`
- [ ] I have added relevant tags if possible for us to better distinguish different PRs

## 🚨 Issue number

> Link this PR to your issue with words like fixed to automatically close the linked issue upon merge
>
> e.g. `fixed #1234`, `closed #1234`, `resolved #1234`

## 📝 What does this PR do?
This PR is used to add installation req for colossal-inference 

## 💥 Checklist before requesting a review

- [ ] I have linked my PR to an issue ([instruction](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue))
- [ ] My issue clearly describes the problem/feature/proposal, with diagrams/charts/table/code if possible
- [ ] I have performed a self-review of my code
- [ ] I have added thorough tests.
- [ ] I have added docstrings for all the functions/methods I implemented

## ⭐️ Do you enjoy contributing to Colossal-AI?

- [ ] 🌝 Yes, I do.
- [ ] 🌚 No, I don't.

Tell us more if you don't enjoy contributing to Colossal-AI.

### CODE REVIEW

File: colossalai/inference/README.md [hunk 1]
```diff
@@ -55,18 +55,25 @@ dependencies
...
 ### Docker
 
-You can use our official docker container as well.
+You can use docker run to use docker container to set-up environment 
+
+```
+# env: python==3.8, cuda 11.6, pytorch == 1.13.1 triton==2.0.0.dev20221202, vllm kernels support, flash-attention-2 kernels support 
+docker pull hpcaitech/colossalai-inference:v2 
+docker run -it --gpus all --name ANY_NAME -v $PWD:/workspace -w /workspace hpcaitech/colossalai-inference:v2 /bin/bash
+
```
Comment (yuanheng-zhao) [inline colossalai/inference/README.md:76]: empty lines

**Review by yuanheng-zhao — APPROVED** [review]

File: colossalai/inference/README.md [hunk 2]
```diff
@@ -55,18 +55,25 @@ dependencies
...
 ```bash
 pytorch= 1.13.1 (gpu)
+cuda>= 11.6 
 transformers= 4.30.2
 triton==2.0.0.dev20221202
-vllm=
-flash-attention=
+# for install vllm, please use this branch to install https://github.com/tiandiao123/vllm/tree/setup_branch
+vllm
+# for install flash-attention, please use commit hash: 67ae6fd74b4bc99c36b2ce524cf139c35663793c
```
Comment (yuanheng-zhao) [inline colossalai/inference/README.md:63]: Are we going to add cmd like `pip install git+https://github.com/...`?

Comment (tiandiao123) [reply colossalai/inference/README.md:63]: we can

Comment (tiandiao123) [reply colossalai/inference/README.md:63]: add it later on .

**Pull request merged**